docker/images/clueso-docker-livy/entrypoint.sh

```diff
@@ -16,4 +17,16 @@ else
   export LIBPROCESS_IP=$HOST
 fi
 
+# wait for Spark master before starting livy
+echo "Waiting for Spark Master to launch on 7077..."
+
+while ! nc -z spark-master 7077 &> /dev/null; do   
```

Comment:
does this need to be a while loop?  can't it just be a condition now?

Reply:
yeah makes more sense to do conditional